whatstk/wparser.py

```diff
@@ -340,6 +340,40 @@ def get_intervention_table_days_hours(data):
     df = df.fillna(0)
     return df
 
+
+def get_intervention_grid_week_hours(data):
+    """
+    Return DataFrame with interventions of all users (columns) for all days (rows)
+
+    Parameters
```

Comment:
Nice docstring! But the listed parameters are not fully in the function's parameters, only 'data' is listed. In fact, 'users' is not even used in this funcion. Am I missing something, maybe?